Debian/Ubuntu: Java Classpath misses /usr/share/jitsi

In Ubuntu 20.04 LTS, after installing Jitsi Desktop as described [here](https://github.com/jitsi/jitsi/issues/793#issuecomment-1534666983), I try to
```
$ jitsi
```
from the Terminal. This gives me
```
exec /usr/bin/java -cp 'lib/*:config/' --add-opens=java.base/jdk.internal.loader=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED -splash:/usr/share/jitsi/splash.gif net.java.sip.communicator.launcher.Jitsi
Error: Could not find or load main class net.java.sip.communicator.launcher.Jitsi
Caused by: java.lang.ClassNotFoundException: net.java.sip.communicator.launcher.Jitsi
```
If I start the Java Runtime Environment directly and append ``/usr/share/jitsi``to the parameter ``cp`` like
```
$ java -cp '/usr/share/jitsi/lib/*:/usr/share/jitsi/config/' --add-opens=java.base/jdk.internal.loader=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED -splash:/usr/share/jitsi/splash.gif net.java.sip.communicator.launcher.Jitsi
```
everything works.

--- a/resources/install/debian/jitsi
+++ b/resources/install/debian/jitsi
@@ -15,7 +15,7 @@ if $SHOW_SPLASH; then
 fi
 
 COMMAND="$(command -v java) \
-  -cp 'lib/*:config/' \
+  -cp ${SCDIR}/lib/*:${SCDIR}/config/ \
   --add-opens=java.base/jdk.internal.loader=ALL-UNNAMED \
   --add-opens=java.base/java.lang=ALL-UNNAMED \
   ${SPLASH_ARG} \

